.github/workflows/link-check.yml

```
@@ -0,0 +1,19 @@
+name: links
+
+on:
+  workflow_dispatch:
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1
```

To avoid supply-chain attacks, we should include by commit hash and not by version.
```suggestion
        uses: lycheeverse/lychee-action@4a5af7cd2958a2282cefbd9c10f63bdb89982d76 # v1.5.1
```